cmd/broker/fanout/main.go

```diff
@@ -54,7 +52,6 @@ var (
 )
 
 type envConfig struct {
-	PodName                string        `envconfig:"POD_NAME" required:"true"`
```

Comment:
This is not used

Reply:
We may want access to POD_NAME to the future. Maybe leave this out since this change is orthogonal?

Reply:
I prefer to add things when we need them, otherwise they just add to the noises. (I can send a separate PR to remove this if you prefer that way)

Reply:
Or if there will be a need for PodName soon, then I will leave it as is :)

Reply:
Normally I would agree, but I believe it would be useful to tag metrics with pod name as discusses on the broker ingress PR and I don't think it makes sense to remove this if we will need to immediately add it back.

Reply:
Added podName back as it may be needed soon.